### DISCUSSION TITLE
Support of Open Charge Point Protocol (OCPP

### DISCUSSION THREAD

**Discussion by HansNetcom** [serkri/SmartEVSE-3#228] [Ideas]
Is there a possibility to build in support for the
Open Charge Point Protocol (OCPP)

**Comment by dingo35** [comment]
You found a backend company that would interface with an OCPP equipped SmartEVSE?

**Reply by HansNetcom** [reply]
No,
The question is if it is possible to interface via OCCP to a backend company

**Comment by dingo35** [comment]
https://gathering.tweakers.net/forum/list_message/75040478#75040478

**Comment by HansNetcom** [comment]
Thnx 👍Op 18 dec 2023 om 20:18 heeft dingo35 ***@***.***> het volgende geschreven:﻿
https://gathering.tweakers.net/forum/list_message/75040478#75040478

**Comment by HansNetcom** [comment]
Hi DingoIndeed, I found a company that is willing to interface with an OCPP equipped Smart EVSE.I could bring you in contact if you want.Kind regardsHans Olislagers Op 18 dec 2023 om 19:19 heeft dingo35 ***@***.***> het volgende geschreven:﻿
You found a backend company that would interface with an OCPP equipped SmartEVSE?
